docs/GOVERNANCE/TEAMS.md

```diff
@@ -0,0 +1,44 @@
+# Teams
+
+A team is a group of people who work together on a specific area to further Organic Maps.
+Team pages list responsibilities, active contributors ("doers") and permissions they hold, as well as links to team-specific docs and resources.
+
+Its not necessary to be a part of a team in order to submit a PR, take part in a discussion or otherwise participate in team's work.
+People listed there are the most active contributors and "go to" persons, so that its clear whom to ask questions, request reviews, etc.
+They could be trusted with extra permissions and accesses necessary for their work (e.g. merge rights).
+
+If you've been contributing actively then feel free to open a PR/issue to add yourself to relevant teams and/or request permissions needed to facilitate your work.
```

Comment:
The most of the teams will be just links to relevant team pages with all the details.
Maybe there could be a short sub-title (like it is now) so that its more clear what the team is about.

For now I've added short responsibility lists so that its easier to see if we missed some important bits or not.